ngraph/python/src/ngraph/ops.py

```diff
@@ -3438,3 +3446,49 @@ def proposal(
     return _get_node_factory().create(
         "Proposal", [class_probs, box_logits, as_node(image_shape)], attrs
     )
+
+
+@nameable_op
+def tensor_iterator(
+    inputs: List[Node],
+    graph_body: GraphBody,
+    slice_input_desc: List[TensorIteratorSliceInputDesc],
+    merged_input_desc: List[TensorIteratorMergedInputDesc],
+    invariant_input_desc: List[TensorIteratorInvariantInputDesc],
+    body_output_desc: List[TensorIteratorBodyOutputDesc],
+    concat_output_desc: List[TensorIteratorConcatOutputDesc],
+    name: Optional[str] = None,
+) -> Node:
+    """
+    Perform recurrent execution of the network described in the body, iterating through the data.
+
+    :param      inputs:                The provided to TensorIterator operator.
+    :param      graph_body:            The graph representing the body we execute.
+    :param      slice_input_desc:      The descriptors describing sliced inputs, that is nodes
+                                       representing tensors we iterate through, processing single
+                                       data slice in one iteration.
+    :param      merged_input_desc:     The descriptors describing merged inputs, that is nodes
+                                       representing variables with initial value at first iteration,
+                                       which may be changing through iterations.
+    :param      invariant_input_desc:  The descriptors describing invariant inputs, that is nodes
+                                       representing variable with persistent value through all
+                                       iterations.
+    :param      body_output_desc:      The descriptors describing body outputs from specified
+                                       iteration.
+    :param      concat_output_desc:    The descriptors describing specified output values through
+                                       all the iterations concatenated into one node.
+    :param      name:                  The optional name for output node.
+
+    :returns:   Node representing TensorIterator operation.
+    """
+
+    attributes = {
+        "body": graph_body.serialize(),
+        "slice_input_desc": [desc.serialize() for desc in slice_input_desc],
+        "merged_input_desc": [desc.serialize() for desc in merged_input_desc],
+        "invariant_input_desc": [desc.serialize() for desc in invariant_input_desc],
+        "body_output_desc": [desc.serialize() for desc in body_output_desc],
+        "concat_output_desc": [desc.serialize() for desc in concat_output_desc],
+    }
+
+    return _get_node_factory().create('TensorIterator', inputs, attributes)
```

Comment:
```suggestion
    return _get_node_factory().create('TensorIterator', as_nodes(*inputs), attributes)
```

Reply:
Is it necessary? `inputs` is has already type marked as: `List[Node]`.